grafana-plugin/src/components/AlertTemplates/AlertTemplatesForm.tsx

```diff
@@ -97,13 +94,10 @@ const AlertTemplatesForm = (props: AlertTemplatesFormProps) => {
   }, [templates]);
 
   const groups = useMemo(() => {
-    const groups: { [key: string]: any } = {};
+    const groups: { [key: string]: Template[] } = {};
```

Comment:
@maskin25 this is the proper typing of `groups` here. This allows us to remove several references to `any` in this file, and also confidently make the change from `==` to `===` equality checking in this file.